README.md

```diff
@@ -43,6 +43,14 @@ If you use Docker for Mac, use `-Dio.jenkins.dev.host` and additional `socat` im
 docker run -d -v /var/run/docker.sock:/var/run/docker.sock -p 2376:2375 bobrik/socat TCP4-LISTEN:2375,fork,reuseaddr UNIX-CONNECT:/var/run/docker.sock
 ```
 
+Connect to Jenkins and login:
+
+Open 127.0.0.1:8080 (by default) in your browser while the server is running.  
```

Comment:
It's not a default address. The default address is actually `DOCKER_HOST:8080` though it becomes a bit more complicated in Docker for Windows/Mac

Reply:
Ok, it seems like it should be a minisection on how to detect/set your factual server address. Will dive into it a bit. Worth doing because this obviously won't be usable otherwise. And plus there is a Dockerfile which in fact requests but ignores your DEV_HOST input.